website/src/templates/index.js

```diff
@@ -197,7 +197,7 @@ class Layout extends React.Component {
                     search={<Search settings={siteMetadata.docSearch} />}
                     alert={nightly ? null : navAlert}
                 >
-                    <Progress key={location.href} />
```

Comment:
I think the idea her was to ensure the progress bar is reset on site changes, which would otherwise not be the case. Can you think of a better way to solve this? Alternatively, I think we probably need to leave this in.

(Sorry, currently battling some Gatsby errors and haven't managed to run the site locally yet from this PR to check!)

Reply:
I tested it, and it's also gone in the final [deploy preview](https://deploy-preview-11967--spacy.netlify.app/) and it is still working. As per [3f00d7d](https://github.com/explosion/spaCy/pull/11974/commits/3f00d7d9386e3ef3299cf60ad78a6cb870970ea8) this was removed to reduce the reliance of the template on `location`.